src/datasources/mockups/ProposalBookingDataSource.ts

```diff
@@ -21,6 +21,9 @@ export default class MockupProposalBookingDataSource
   get(id: number): Promise<ProposalBooking | null> {
     throw new Error('Method not implemented.');
   }
+  delete(proposalPk: number): Promise<ProposalBooking | null> {
+    throw new Error('Method not implemented.');
```

Comment:
I see there are not many unit tests. 
Do you think it would be beneficial to add?

Reply:
No I think scheduler backend is missing unit tests in general because that was a decision at the beginning that they are not so beneficial. I added this method just to fix the linting errors but we can remove that file completely because it is not used at all.